app/controllers/stripe-setup/responsible-person/responsible-person-validations.test.js

```diff
@@ -0,0 +1,71 @@
+'use strict'
```

Comment:
Do we name any other tests `.test.js`? I haven't seen this before - I think normally `spec` is used.

Reply:
We have … a few different conventions, including `.test.js`